### DISCUSSION TITLE
Target_link on /oidcRedirect?error

### DISCUSSION THREAD

**Discussion by hf409** [OpenIDC/mod_auth_openidc#631] [Q&A] [answered]
Hi,

is it possible to retrieve the original URL the user requested in case of error? 
It is send as parameter to oidcRedirect?target_link_uri.
As result I get a state and and error parameter.
I guess the target_link_uri is part of the mod_auth_openidc_state cookie. 
Is there an easy way to get the information?
Or can I send an extra parameter that is bounced back?

Thanks for any help,
Heike

**Comment by zandbelt** [comment, marked as answer]
no, it's not possible to get hold of that value

**Comment by hf409** [comment]
Thanks for the answer.
So, I will use several vhosts to keep track of it. Not the elegant way, but I guess this will work.
